sdk/open_api/api/market_data_api.py

```
@@ -1208,6 +1209,306 @@ def _get_market_spot_executions_serialize(
+        self,
+        symbol: Annotated[str, Field(strict=True, description="Trading symbol (e.g., BTCRUSDPERP)")],
+        start_time: Annotated[Optional[Annotated[int, Field(strict=True, ge=0)]], Field(description="Return results after this sequence number (for pagination)")] = None,
+        end_time: Annotated[Optional[Annotated[int, Field(strict=True, ge=0)]], Field(description="Return results before this sequence number (for pagination)")] = None,
+        _request_timeout: Union[
+            None,
+            Annotated[StrictFloat, Field(gt=0)],
+            Tuple[
+                Annotated[StrictFloat, Field(gt=0)],
+                Annotated[StrictFloat, Field(gt=0)]
+            ]
+        ] = None,
```

Ruff (0.15.5)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)